### PR TITLE
Opsd renewable ppls

### PR DESCRIPTION
**Pull request by FabianHofmann**
Builds on and supersedes #203 

Before merging, I have to enable the new ppm version in conda-forge

## Checklist

- [x] I tested my contribution locally and it seems to work fine.
- [x] Code and workflow changes are sufficiently documented.
~- [] Newly introduced dependencies are added to `environment.yaml` and `environment.docs.yaml`.~
- [x] Changes in configuration options are added in all of `config.default.yaml`, `config.tutorial.yaml`, and `test/config.test1.yaml`.
- [x] Changes in configuration options are also documented in `doc/configtables/*.csv` and line references are adjusted in `doc/configuration.rst` and `doc/tutorial.rst`.
- [x] A note for the release notes `doc/release_notes.rst` is amended in the format of previous release notes.
- [x] Updated ppm on conda-forge

### CODE REVIEW

**Comment by FabianHofmann** [comment]
this pr is ready however, we have to wait until the new ppm version is available in conda-forge

**Comment by fneum** [comment]
> until new ppm version is available in conda-forge

Isn't that more or less instantaneous when you release a new version?

**Comment by FabianHofmann** [comment]
not so sure, I think it take about an hour or two till they update from the feedstocks

**Comment by fneum** [comment]
Ok, yes then no worries about this very small delay!

File: envs/environment.yaml [hunk 1]
```diff
@@ -25,7 +25,7 @@ dependencies:
   - yaml
   - pytables
   - lxml
-  - powerplantmatching>=0.4.3
+  - powerplantmatching>=0.4.7
```
Comment (fneum) [inline envs/environment.yaml:28]:
```suggestion
  - powerplantmatching>=0.4.8
```

**Pull request merged**